src/main/java/org/folio/linked/data/service/resource/impl/ResourceServiceImpl.java

```diff
@@ -101,6 +101,8 @@ private Resource getResource(Long id) {
   private Resource saveNewResource(ResourceRequestDto resourceDto, Resource old) {
     var mapped = resourceDtoMapper.toEntity(resourceDto);
     metadataService.ensure(mapped, old.getFolioMetadata());
+    mapped.setCreatedDate(old.getCreatedDate());
+    mapped.setVersion(old.getVersion() + 1);
```

Comment:
What about setting createdBy/updatedBy?

Reply:
thought it was out of the scope of the story, let me check how we can do this using AuditorAware.